zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/ReadOnlyZooKeeperServer.java

```diff
@@ -190,23 +190,23 @@ public long getServerId() {
     }
 
     @Override
-    public synchronized void shutdown() {
+    public synchronized void shutdown(boolean fullyShutDown) {
         if (!canShutdown()) {
+            super.shutdown(fullyShutDown);
```

Comment:
Should this line be here? It seems like we're calling `super.shutdown(fullyShutDown)` twice.

Reply:
```suggestion
```
Thanks! This should indeed not be here.